Document GitHub Local Actions integration

SanjulaGanepola opened the pull request.

SanjulaGanepola committed:
Document GitHub Local Actions integration

Signed-off-by: Sanjula Ganepola <sanjulagane@gmail.com>

SanjulaGanepola committed:
Fix wording

Signed-off-by: Sanjula Ganepola <sanjulagane@gmail.com>

SanjulaGanepola commented:
@cplee This PR is to document the extension in the docs as we discussed

cplee commented on src/integrations.md:
```diff
@@ -5,3 +5,4 @@
 - [Gitea Actions runner](https://gitea.com/gitea/act_runner)
 - [github-act-runner](https://github.com/ChristopherHX/github-act-runner)
 - [Gradle Act plugin](https://github.com/pshevche/gradle-act-plugin)
+- [GitHub Local Actions](https://sanjulaganepola.github.io/github-local-actions-docs/)
```
Can you update the link to be clear that this is a VSCode extension?

ChristopherHX commented on src/introduction.md:
```diff
@@ -9,6 +9,13 @@ Run your [GitHub Actions](https://developer.github.com/actions/) locally! Why wo
 
+---
+
+> ✨ **Now Run GitHub Actions Locally Directly From VS Code!** ✨  
```
I'm not a fan of emojis, only speaking for myself who can't merge this.

I think this fits better into the docs, your slogan reads more like advertising your product name of your extension than describing the benefit.
```suggestion
> ✨ **Now Manage and Run Act Directly From VS Code!** ✨  
```

_Casey has the last word about this, it's his website domain that hosts the content and I don't know his opinion_.  Eventually he would just take it as is.

cplee replied:
Let's remove emojis for consistency with rest of docs.

ChristopherHX reviewed (commented):
_It's silent here_

SanjulaGanepola committed:
Remove emojis, update link, and update slogan

Signed-off-by: Sanjula Ganepola <Sanjula.Ganepola@ibm.com>

SanjulaGanepola commented:
@cplee I addressed the PR comments. I also opened a PR in the main repo to update the README as well if the change here is good: https://github.com/nektos/act/pull/2588

cplee commented on src/integrations.md:
```diff
@@ -5,3 +5,4 @@
 - [Gitea Actions runner](https://gitea.com/gitea/act_runner)
 - [github-act-runner](https://github.com/ChristopherHX/github-act-runner)
 - [Gradle Act plugin](https://github.com/pshevche/gradle-act-plugin)
+- [GitHub Local Actions](https://marketplace.visualstudio.com/items?itemName=SanjulaGanepola.github-local-actions)
```
```suggestion
- [GitHub Local Actions for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=SanjulaGanepola.github-local-actions)
```

SanjulaGanepola replied:
@cplee I see what you mean now. Can I go ahead with this change but keep it at the previous link I had? Or this link works as well.

cplee replied:
Whatever link you want works for me - so long as the text of the link is clear what it is.

SanjulaGanepola replied:
Great, I updated to the previous link. The PR to update the README uses the same link so it makes sense to keep it the same.

SanjulaGanepola committed:
Update src/integrations.md

Co-authored-by: Casey Lee <cplee@nektos.com>

SanjulaGanepola committed:
Update link

Signed-off-by: Sanjula Ganepola <Sanjula.Ganepola@ibm.com>
